README.md

```diff
@@ -89,6 +89,12 @@ To Start a Docker Compose Instance for Testing / Debugging
     
    You can start multiple compose instances on the same project as the plugin generates a unique name for each instance.
    
+   To use the static host ports instead of the Docker dynamically assigned host ports use the following command:
+   
+    dockerComposeUp -useStaticPorts
```

Comment:
@edward720 and I discussed this. I'd like all new parameters to include a "-". I'd like to go back and add the "-" to skipBuild and skipPull but don't want to break backwards compatibility. We might make this change as part of a major version release.